CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 15.19.0
+
+_released `2024-11-25`_
+
+For optimal performance, an optional argument has been added to [z.dehydrate](https://docs.zapier.com/platform/build-cli/overview#dehydration) to allow devs to control the caching duration on hydrators.
+
+Also, with the new `meta` field object added to the [FieldSchema](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#fieldschema) object, devs have more flexibility storing field-specific context which may be obtained when constructing dynamic needs, and then making that context available during actual action execution in the perform method.
+
+Lastly, the [zapier scaffold](https://github.com/zapier/zapier-platform/blob/main/packages/cli/docs/cli.md#scaffold) now supports TypeScript (TS) applications. It automatically detects TS and JS projects if the `--entry` flag isn't provided, so that it can generate and edit the right files in the right places, given TS projects tend to keep their files in ./src.
+
+### cli
+
+- :nail_care: Add support for TS on the `zapier scaffold` command ([#884](https://github.com/zapier/zapier-platform/pull/884))
+
+### core
+
+- :tada: Add support for providing the cache expiration for hydrators ([#918](https://github.com/zapier/zapier-platform/pull/918))
+- :scroll: Add `timezone` and `withSearch` fields to the `bundle.meta` object ([#922](https://github.com/zapier/zapier-platform/pull/922))
+- :scroll: Update the `BeforeRequestMiddleware` typings to require `url` in the request object ([#924](https://github.com/zapier/zapier-platform/pull/924))
+
+### schema
+
+- :tada: Add `meta` field support for the `FieldSchema` ([#883](https://github.com/zapier/zapier-platform/pull/883))
+
+### schema-to-ts
+
+None!
+
+### misc
+
+- :hammer: Bump elliptic from 6.5.7 to 6.6.0 ([#911](https://github.com/zapier/zapier-platform/pull/911))
+- :tada: Add schema-to-ts as part of yarn workspace ([#913](https://github.com/zapier/zapier-platform/pull/913))
```

Comment:
@kola-er I believe this MR (#913) is not part of 15.19.0; it was merged into the `zapier-platform-major-release-16.0.0-dev` branch (not `main`)

Reply:
Thanks, removed it.